src/server/api/controllers/members.controller.js

```diff
@@ -34,6 +47,29 @@ const getAllMembers = async (userId, boardId) => {
   }
   return allMembers;
 };
+
+const addMember = async (userId, boardId, memberId) => {
+  if (
+    !Number.isInteger(Number(userId)) ||
+    !Number.isInteger(Number(boardId)) ||
+    !Number.isInteger(Number(memberId))
+  ) {
+    throw new InvalidIdError('Id should be an integer');
+  }
+  const isOwner = await userIsOwner(userId, boardId);
+  if (isOwner.length === 0) {
+    throw new UnauthorizedError(`Only owner can add a member to a board `);
+  }
```

Comment:
This condition duplicates logic in the `userIsOwner` function. That function already throws an error so you don't need to do so here again.

Reply:
was not sure where it belongs, so left at both places for one to be plucked out. Removed this. Thanks, @Midnighter! ⭐